examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
```

Comment:
Ya, this is not a long term solution. I think the remaining thing to figure out still with #8410 is how to treat more complex things that are part of the information architecture like the scaffold and app bar and how they fit in the gallery.